packages/feedback-service/src/resolver.ts

```diff
@@ -141,6 +141,7 @@ export const FeedbackResolver = {
           } );
         } )
         .catch( ( err: any ) => {
+          err.options.auth = '*******';
```

Comment:
Do we really want this entire error object(err) to transferred to the front end? Can we just extract error message out of it and transferred that only?

Reply:
Updated as per suggestion.